themes/hugo-theme-learn/layouts/_default/sitemap.xml

```diff
@@ -0,0 +1,32 @@
+{{ printf "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"yes\" ?>" | safeHTML }}
```

Comment:
Why is this a Go template line?

Reply:
I am just grab this [default template](https://gohugo.io/templates/sitemap-template/#hugos-sitemapxml) from recommendations on Hugo website. The [safeHTML](https://gohugo.io/functions/safehtml/) function is using for correct generation for HTML tags. There are attributes here with escaping for double quotes and they needs to be also generated correctly. Suppose, all above is also needed for XML tags as well.